src/main/java/bio/terra/service/dataset/flight/ingest/IngestUtils.java

```diff
@@ -239,10 +241,34 @@ public static long countBulkFileLoadModelsFromPath(
             cloudEncapsulationId,
             fileRefColumns,
             errors)) {
+      nodesStream
+          .takeWhile(bulkLoadFileModel -> errors.size() <= maxBadLoadFileLineErrorsReported)
+          .forEach(
+              loadFileModel -> {
```

Comment:
Note that `{` `}` isn't necessary in a lambda expression if there's only one statement, as is true in this case.

Reply:
I think it's needed with this multi-line logic.

Reply:
You're right, it can be omitted if the content is a single expression but not if it's a single statement.